### PR TITLE
[DOCS-6372] AGS Community

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by anxumalo — COMMENTED** [review]
For consistency, I'd also suggest deleting the following page & update the config (so it's similar to ACS Community) as it serves no real purpose: 
- `governance-services/community/support/index.md`

File: governance-services/community/develop/index.md [hunk 1]
```diff
@@ -8,9 +8,7 @@ Governance Services provides two APIs, the GS Core API and the GS Security Marks
 
 The APIs are designed for you to create remote clients to manage the {% include tooltip.html word="fileplan" text="File Plan" %}. You can easily explore and test the endpoints using the Governance Services API Explorer.
 
-You can download the API Explorer from:
-
-* [Alfresco Community RM ReST API Explorer](https://download.alfresco.com/cloudfront/release/community/RM/3.5.a/alfresco-governance-services-community-rest-api-explorer-3.5.a.war)
+You can download the API Explorer `alfresco-governance-services-community-rest-api-explorer-11.140.war` from [Hyland Community](https://community.hyland.com/){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply governance-services/community/develop/index.md:11]: Updated.

File: governance-services/community/install/zip.md [hunk 2]
```diff
@@ -4,10 +4,12 @@ title: Install using the distribution ZIP
 
 Governance Services is installed by applying two AMP files to an existing Alfresco Community Edition installation.
 
-The Governance Services distribution zip file contain the following AMP files:
+Download the Alfresco Governance Services distribution zip from [Hyland Community](https://community.hyland.com/){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply governance-services/community/install/zip.md:7]: updated

**Comment by oliveralfresco2018** [comment]
Adjusted the comments plus adjusted the AGS Ent one also.

Cheers

**Review by gravitonian — CHANGES_REQUESTED** [review]
In http://localhost:4000/governance-services/community/install/docker/: you refer to 
 services:
     alfresco:
         #image: alfresco/alfresco-content-repository-community:7.0.0
         image: alfresco/alfresco-governance-repository-community:3.5.a

Is version 3.5.a correct???

**Comment by oliveralfresco2018** [comment]
Fixed that up thanks

**Review by gravitonian — APPROVED** [review]

**Pull request merged**